
The following have been reloaded with a version change:
  1) python/3.8-anaconda3 => python/3.7.7


###  s 168 59523776 9999994368 0 45609.440728312446 11111053623
###  s 168 59523776 9999994368 1 56813.412950152444 11111053623
###  s 168 59523776 9999994368 2 56890.86350656691 11111053623
###  s 168 59523776 9999994368 3 55641.48292371205 11111053623
###  s 168 59523776 9999994368 4 58511.049060594465 11111053623
ERROR:  One or more process (first noticed rank 13) terminated with signal 9

real	5m41.761s
user	0m0.525s
sys	0m0.136s

------------------------------------------------------------
Sender: LSF System <lsfadmin@batch4>
Subject: Job 2502537: <cylonrun-s-4> in cluster <summit> Done

Job <cylonrun-s-4> was submitted from host <login4> by user <gregorvl> in cluster <summit> at Wed Oct 12 16:00:00 2022
Job was executed on host(s) <1*batch4>, in queue <batch>, as user <gregorvl> in cluster <summit> at Sat Oct 15 13:24:20 2022
                            <42*f17n02>
                            <42*f17n03>
                            <42*f17n04>
                            <42*f17n05>
</ccs/home/gregorvl> was used as the home directory.
</ccs/home/gregorvl/cylon/summit/scripts/strong_scaling> was used as the working directory.
Started at Sat Oct 15 13:24:20 2022
Terminated at Sat Oct 15 13:30:33 2022
Results reported at Sat Oct 15 13:30:33 2022

The output (if any) is above this job summary.

